client/components/theme/index.jsx

```diff
@@ -123,7 +123,7 @@ export class Theme extends Component {
 					aria-label={ this.props.theme.name }
 					title={ this.props.theme.description }
 					className="theme__active-focus"
-					href={ this.props.screenshotClickUrl || '#' }
```

Comment:
Apparently, this fallback was introduced by https://github.com/Automattic/wp-calypso/commit/100f3cde7dbedee5e7bdca6875293a84b60d90fb for a11y reasons, so we'll also need to verify we're not introducing a regression against that...